### PR TITLE
Add support for viainvest

### PR DESCRIPTION
**Pull request by AlexanderLill**
This adds support for parsing viainvest account statements. Withdrawals are currently not supported, only deposits and interest payments.

Currently I can't get the tests to run on my environment. @ChrisRBe is there a commandline command to run all tests?

Edit: I just saw that also there are formatting issues, is there a command to run this locally or a nice way to show how the formatting should look like? :)

### CODE REVIEW

**Comment by AlexanderLill** [comment]
Any feedback on how to overcome the pipeline checks is very welcome :)

**Comment by ChrisRBe** [comment]
Hi @AlexanderLill,

Thanks for the PR first :) 

Depends on how you set up your local environment. I wanted to update the development section for some time already, but always find excuses not to do it.


Here goes (this works for me under Windows, should also work in other environments):

Some time ago I migrated most of the dev env from the requirements.txt file to pipenv. See also documentation here: https://pypi.org/project/pipenv/

pipenv basically sets up a virtualenv for you including black, pre-commit, flake, pylint, etc. 
inside your environment I then enabled pre-commit, see documentation here: https://pre-commit.com.

If you don't want to do that (which is fine :)), the (virtual) env that is installed by both requirements.txt and Pipfile should deploy a local version of black. See https://github.com/psf/black.

Inside your python environment you should be able to run black simply by invoking it via `black ./src`. I believe it will use the config from the repository and re-format the code accordingly.

Running unit tests locally should also be as simple as running it via `pytest` from your local environment. I do think that it will also use the provided config from this repos configuration.

File: src/Statement.py [hunk 1]
```diff
@@ -92,6 +93,23 @@ def get_currency(self):
             return self._statement[self._config.get_booking_currency()]
         else:
             return "EUR"
+    
+    @staticmethod
+    def _parse_value(value):
```
Comment (ChrisRBe) [inline src/Statement.py:98]: doc string missing :)

**Review by ChrisRBe — CHANGES_REQUESTED** [review]

File: src/Statement.py (hunk 1 above)
Comment (AlexanderLill) [reply src/Statement.py:98]: Thanks, fixed :)

Comment (ChrisRBe) [reply src/Statement.py:98]: Nice. Thank you. Almost perfect :). The value param description is missing

Comment (AlexanderLill) [reply src/Statement.py:98]: True! Too bad that black does not check for this. I also found nothing in the documentation about adding this check :(

**Comment by ChrisRBe** [comment]
I'm also looking into the codeclimate failure. Looks like codeclimate thinks that it should also consider the test code for overall coverage. Which is eh... not what we want. 

I probably have to update one additional config file to fix this properly.

**Comment by AlexanderLill** [comment]
@ChrisRBe  from my side I don't see anything left to fix, even though the codeclimate check is failing. When I click on details it shows no issues, very strange :(

File: src/test/test_statement.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""
+Unit test for the p2p statement class
+
+Copyright 2021-12-12 AlexanderLill
+"""
+import unittest
+
+from Statement import Statement
```
Comment (ChrisRBe) [inline src/test/test_statement.py:9]: I stumbled upon an import error in my local environment for some reason. Could you please update it to?

```suggestion
from src.Statement import Statement
```

Thank you.

**Review by ChrisRBe — CHANGES_REQUESTED** [review]
Looks good. one more change :)

**Comment by ChrisRBe** [comment]
> @ChrisRBe from my side I don't see anything left to fix, even though the codeclimate check is failing. When I click on details it shows no issues, very strange :(

Yea, I see the same .. I believe it's caused by coverage also reporting on the unit test files.. I have an open PR #460 to address this hopefully. We might have to update this PR via merge and run another check.

**Comment by ChrisRBe** [comment]
bors merge

**Comment by ChrisRBe** [comment]
bors merge

**Pull request merged**